src/code.cloudfoundry.org/loggregator/plumbing/conversion/tov2.go

```diff
@@ -83,12 +79,14 @@ func initTags(v1e *events.Envelope, v2e *v2.Envelope, oldTags map[string]string,
 		if v2e.Tags == nil {
```

Comment:
Why pass in `oldTags` (which is `v1e.GetTags()`) and also use `v1e.Tags` in the assignment above? I would think either receive `oldTags` or don't.